client/lib/geocoding/test/index.js

```diff
@@ -15,6 +16,8 @@ import { geocode } from '../';
 const TEST_ADDRESS = '1600 Amphitheatre Parkway, Mountain View, CA';
 
 describe( 'geocoding', () => {
+	useNock();
```

Comment:
I forgot that `useMock` need to work this way to make sure it plays nicely with `useFilesystemMocks`.